js/app.js

```diff
@@ -0,0 +1,107 @@
+console.log('yo')
+// combine objects, arrays, and functions
+const curbArray = [
+	characters: {
+		name1: 'jeff',
+		name2: 'larry',
+		name3: 'susie',
+		name4: 'cheryl',
+		name5: 'funkhouser'
+	}
+]
+console.log(curbArray.name5)
+
+
+
+
+let curbObj = {
+	funny: true,
+	characters: ['larry', 'jeff', 'cheryl', 'susie', 'funkhouser']
+};
+console.log(curbObj.characters[4])
+
+let curb2 = [
+	characters: ['larry', 'jeff', 'cheryl', 'susie', 'funkhouser']
+	seasons: ['season 1', 'season 2', 'season 3', 'season 4', 'season 5', 'season 6', 'season 7', 'season 8', 'season 9'  ]
+
+]
+console.log(curb2[1][5])
+
+// combine objects, arrays, and functions
+
+// combine objects, arrays, and functions at more than 1 level
+
+
+
+
+
+// combine objects, arrays, and functions at more than 1 level
+
+
+// Create callback
+
+
+
+// create callback
+
+
+// indentation
+if(true){
+  
+  const a = 2 + 2;
+  
+console.log(a);
+ }
+
```

Comment:
line 53 should be indented right more
} on line 54 should be all the way left

-0.5